.travis.yml

```diff
@@ -1,15 +1,20 @@
-language: c
-sudo: false
+language: c++
+# Need sudo for trusty.
+sudo: required
+# Trusty gives us GCC 4.8 which supports c++11.
+dist: trusty
```

Comment:
so we should be able to get modern gcc w/o `sudo` with `addons: apt:` see http://genbattle.bitbucket.org/blog/2016/01/17/c++-travis-ci/